dev.sh

```diff
@@ -7,15 +7,38 @@ PORT=${DOCKERPORT:-80}
 DB_READY=0
 TRIES=0
 MAX_TRIES=12
+export SOLR=4.10.4
 
 docker stop open-oni-dev || true
 docker rm open-oni-dev || true
 
+# start_container $1 = name of container, $2 = container running status
+start_container () {
```

Comment:
I dunno - maybe leave it as-is and do that later if we find a need for it?  I am not as close to the situation as you, though.  If you feel like it would ease things in the future, go for it.